pkg/models/live.go

```diff
@@ -17,9 +17,10 @@ type ChannelHandler interface {
 	OnPublish(c *centrifuge.Client, e centrifuge.PublishEvent) ([]byte, error)
 }
 
-// ChannelHandlerProvider -- this should be implemented by any core feature
-type ChannelHandlerProvider interface {
-	// This is called fast and often -- it must be synchrnozed
+// ChannelNamespaceHandler should be implemented by all core features.
+type ChannelNamespaceHandler interface {
```

Comment:
I suggest we rename the interface as follows, since the current name doesn't reflect what it does:
```suggestion
// ChannelHandlerFactory should be implemented by all core features.
type ChannelHandlerFactory interface {
```